pkg/tfshim/sdk-v2/provider.go

```diff
@@ -62,6 +62,7 @@ func NewProvider(p *schema.Provider, opts ...providerOption) shim.Provider {
 		tf:   p,
 		opts: opts,
 	}
+
```

Comment:
Nit: remove whitespace change to simplify the diff.

Reply:
Mb, thanks for spotting.